packages/sdk-react/src/components/payables/Payables.tsx

```diff
@@ -186,7 +186,8 @@ const PayablesBase = ({
           toast.promise(
             payableUploadFromFileMutation.mutateAsync({
               file,
-              file_type: 'payables',
+              // TODO why is this file_type was removed in 2024-01-31?
```

Comment:
Docs say that file_type indeed was removed. 
https://docs.monite.com/api/payables/post-payables-upload-from-file#request
There is no need in TODO here.